uniffi_bindgen/src/interface/record.rs

```diff
@@ -172,7 +200,7 @@ mod test {
         assert_eq!(record.fields().len(), 1);
         assert_eq!(record.fields()[0].name(), "field");
         assert_eq!(record.fields()[0].type_().canonical_name(), "u32");
-        assert!(!record.fields()[0].required);
+        assert!(!record.fields()[0].required());
```

Comment:
We can no longer access the fields directly because they're hidden behind a pointer.